front-end/src/components/SetSecurityLevel.js

```diff
@@ -18,44 +17,31 @@ const useStyles = makeStyles((theme) => ({
 
 export default function SetSecurityLevel() {
   const classes = useStyles();
-  const [age, setAge] = React.useState('');
-  const [open, setOpen] = React.useState(false);
 
   const handleChange = (event) => {
-    setAge(event.target.value);
-  };
-
-  const handleClose = () => {
-    setOpen(false);
-  };
-
-  const handleOpen = () => {
-    setOpen(true);
+    switch (event.target.value) {
+      case '1.0':
+        window.location.href = 'http://35.224.40.226:5000/#/';
```

Comment:
I noticed that it uses the same window to route. Is there a way to set options to set` target="_blank"` and `rel="noopener noreferrer"` so that it will open in a new tab/window? Otherwise the user won't be able to use the dropdown in the other versions.

https://stackoverflow.com/questions/15551779/open-link-in-new-tab-or-window

Maybe instead of having a `handleOpen` method we can the menu items as anchor `<a>`  tags and we could set the target and rel then, but if there's a way to set those here that would probably be path of least resistance.

Reply:
Yeah good call, apparently `window.open` has a `target` field so I just changed it to that. Seems to be working for me locally now. (for opening in new tabs)